README.rst

```diff
@@ -118,6 +118,7 @@ It also has the below extra options:
 
 * ``-E`` / ``--skip-errors`` - Don’t exit non-zero for errors from Black (normally syntax errors).
 * ``--rst-literal-blocks`` - Also format literal blocks in reStructuredText files (more below).
+* ``--check`` - [Similar behaviour to Black's check flag.](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#check)
```

Comment:
Please write a description of the option rather than just link and say "it's like that". The link is still useful though, but don’t link a whole sentence, just a couple words.

Also alphabetize by moving to line 119.

Reply:
Should be fixed in f01c5ab

Do tell me if you're unhappy with the phrasing.